cmd/admin-router.go

```diff
@@ -181,7 +181,9 @@ func registerAdminRouter(router *mux.Router, enableConfigOps, enableIAMOps bool)
 				adminRouter.Methods(http.MethodPut).Path(adminVersion+"/set-bucket-quota").HandlerFunc(
 					httpTraceHdrs(adminAPI.PutBucketQuotaConfigHandler)).Queries("bucket", "{bucket:.*}")
 			}
-			// Bucket replication operations
+		}
+		if env.Get(envDataUsageCrawlConf, config.EnableOn) == config.EnableOn {
+			// Bucket replication/lifecycle operations
```

Comment:
ok, will confirm with AB whether we can drop tiering in FS mode, then update the PR accordingly

Reply:
IMO FS mode should be left alone because of complications this can bring in, because we don't do replication while we still have crawler in FS mode. 

There are no serious deployments in FS mode which warrant this - it simply makes it complicated.

Reply:
You will have to explain why versioning won't work with ILM transition etc in FS mode.

Reply:
just checked with AB, he thinks it makes sense to support transition in FS mode since ILM is already a supported feature.His suggestion was to separate into 2 PR's if that would make it easier.

Reply:
My problem is increasing the scope of FS mode @poornas it really doesn't make sense to me. While we support ILM in FS we don't support all features of ILM i.e DeleteMarker, NoCurrentVersion etc.

No one will ever use it and is asking for it. I would say the first step add this for erasure coding and stabilize it and add it for FS mode if needed in the future.

Reply:
ok, I will factor FS out of this PR.